pennylane/devices/default_tensor.py

```diff
@@ -144,24 +144,105 @@ def accepted_observables(obs: qml.operation.Operator) -> bool:
 @simulator_tracking
 @single_tape_support
 class DefaultTensor(Device):
-    """A PennyLane device to perform tensor network operations on a quantum circuit using
+    """A PennyLane device to perform tensor network simulations of quantum circuits using
     `quimb <https://github.com/jcmgray/quimb/>`_.
 
+    This device is designed to simulate large-scale quantum circuits using tensor networks. For small circuits, other devices like ``default.qubit`` may be more suitable.
+
+    The backend uses the ``quimb`` library to perform the tensor network operations, and different methods can be used to simulate the quantum circuit.
+    Currently, only the Matrix Product State (MPS) method is supported, based on the ``quimb``'s ``CircuitMPS`` class.
+
+    This device does not currently support finite shots, derivatives, or vector-Jacobian products.
```

Comment:
To me, this says that JVPs _are_ supported. Perhaps this is a better way to phrase it?

```suggestion
    This device does not currently support finite shots or differentiation.
```

Reply:
I agree (it is also more concise)